app/uk/gov/hmrc/agentepayeregistration/binders/Bindings.scala

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
we have some previous examples of putting binders into controller/UrlBinders, maybe better stick with it: https://github.com/hmrc/agent-mapping/blob/0ba136e00eebb556baea23d94c5a82a434744df5/app/uk/gov/hmrc/agentmapping/controller/UrlBinders.scala, https://github.com/hmrc/agent-subscription/blob/f869fd7a1d540dc2efd1f947cdd310c2e7a93fa3/app/uk/gov/hmrc/agentsubscription/controllers/UrlBinders.scala